ios/FluentUI/Vnext/Divider/Divider.swift

```diff
@@ -64,38 +63,38 @@ public struct FluentDivider: View {
                                   leading: tokens.padding,
                                   bottom: 0,
                                   trailing: tokens.padding))
-            .designTokens(tokens,
-                          from: theme,
-                          with: windowProvider)
+            .resolveTokens(self)
+            .resolveTokenModifier(self, value: state.orientation)
     }
+
+    public func overrideTokens(_ tokens: DividerTokens?) -> FluentDivider {
+        state.overrideTokens = tokens
+        return self
+    }
+
+    var tokens: DividerTokens { state.tokens }
+    @Environment(\.fluentTheme) var fluentTheme: FluentTheme
+    @ObservedObject var state: MSFDividerStateImpl
 }
 
 /// Properties available to customize the Divider.
-class MSFDividerStateImpl: NSObject, ObservableObject, MSFDividerState {
+class MSFDividerStateImpl: NSObject, ObservableObject, ControlConfiguration, MSFDividerState {
+    @Published var overrideTokens: DividerTokens?
+    @Published var tokens: DividerTokens
+    var defaultTokens: DividerTokens {.init()}
+
     @Published var color: UIColor?
```

Comment:
Now that the divider's color can be set via `DividerTokens`, do we still need a separate property here?